documentation/app/styles/components/page-header.scss

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ */
+
+.doc-PageHeader {
```

Comment:
I was actually trying to use BEM, but it's been a while. Though, I was following this pattern for it:
- The Page Header is the component so it's the block `doc-PageHeader` I used title case here because if you did snake case it would be hard to differentiate the block and the element.
- The title (down below) would be an element of the block so `doc-PageHeader-title` I don't think the title satisfies the definition of a modifier, "Flags on blocks or elements. Use them to change appearance, behavior or state." so I left it as an element.

This is what I was referencing: https://getbem.com/naming/

Reply:
I'm ok with any approach we choice, and I agree that the approach makes it more obvious, what is component itself and what are the elements. 👍 
Design systems follows approach, when `doc-content-principles` - page, `doc-content-principles__description` -page partial. But we don't have some rule to follow what design system follows :)